File: sdkwork-app-sdk-go/types/onboarding_page_vo.go
package types


type OnboardingPageVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	PageId string `json:"pageId"`
	Title string `json:"title"`
	Description string `json:"description"`
	Images []string `json:"images"`
	Buttons []string `json:"buttons"`
	BackgroundColor string `json:"backgroundColor"`
	TextColor string `json:"textColor"`
	IsActive bool `json:"isActive"`
	Order int `json:"order"`
}

File: sdkwork-app-sdk-csharp/Models/OnboardingPageVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class OnboardingPageVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? PageId { get; set; }
        public string? Title { get; set; }
        public string? Description { get; set; }
        public List<string>? Images { get; set; }
        public List<string>? Buttons { get; set; }
        public string? BackgroundColor { get; set; }
        public string? TextColor { get; set; }
        public bool? IsActive { get; set; }
        public int? Order { get; set; }
    }
}
